.github/workflows/security-audit.yml

```
@@ -0,0 +1,266 @@
+            console.log('Security issue already exists, skipping creation');
+          }
+
+  # 发送通知到 Slack/Teams (可选)
+  notify:
+    name: Send Security Notification
+    runs-on: ubuntu-latest
+    needs: [security-audit, security-summary]
+    if: failure() && github.event_name == 'schedule'
+    
+    steps:
+    - name: Send Slack Notification
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)